modules/afsocket/afsocket-grammar.ym

```diff
@@ -461,24 +462,11 @@ source_afnetwork_option
 
 source_afsocket_transport
 	: afsocket_transport
-        | KW_TRANSPORT '(' string
+        | KW_TRANSPORT '(' string ')'
           {
-            Plugin *p;
-            gint context = LL_CONTEXT_SERVER_PROTO;
-
-            p = cfg_find_plugin(configuration, context, $3);
-            if (p)
-              {
-                /* for transports with grammar */
-                if (p->parser)
-                  {
-                    LogProtoServerFactory *sf = cfg_parse_plugin(configuration, p, &@3, last_proto_server_options);
-                    ((AFSocketSourceDriver *) last_driver)->proto_factory = sf;
-                  }
-              }
             transport_mapper_set_transport(last_transport_mapper, $3);
+            free($3);
```

Comment:
I think the reason for this code here was ALTP which had configuration options to parse here.

Reply:
Thanks for letting me know. I have removed this part of the PR as it's not yet fully clear if the code there is still necessary then.